src/clustRviz_base.h

```diff
@@ -7,6 +7,7 @@
 
 #define CLUSTRVIZ_STATUS_UPDATE_TIME_SECS 0.1 // Every 0.1s
 #define CLUSTRVIZ_STATUS_WIDTH_CHECK 20 // Every 20 status updates * 0.1s => every 2s
+#define EXACT_STOP_PRECISION 1e-10 //Stop when cellwise diff between iters < val
```

Comment:
Can we "namespace" this like the other `#define`s? `CLUSTRVIZ_EXACT_STOP_PRECISION`